FakeSyncManager.java won't build.

/home/leonardo/Projects/Buendia/direct/client/app/src/androidTest/java/org/projectbuendia/client/FakeSyncManager.java:41: error: startFullSync() in FakeSyncManager cannot override startFullSync() in SyncManager
    @Override public void startFullSync() {
                          ^
  overridden method is static
/home/leonardo/Projects/Buendia/direct/client/app/src/androidTest/java/org/projectbuendia/client/FakeSyncManager.java:41: error: method does not override or implement a method from a supertype
    @Override public void startFullSync() {
    ^

--- a/app/src/main/java/org/projectbuendia/client/sync/SyncManager.java
+++ b/app/src/main/java/org/projectbuendia/client/sync/SyncManager.java
@@ -83,7 +83,7 @@ public class SyncManager {
     }
 
     /** Starts a full sync as soon as possible. */
-    public static void startFullSync() {
+    public void startFullSync() {
         SyncAccountService.startFullSync();
     }
 

